restapi/src/main/java/io/stargate/web/docsapi/service/util/DocsApiUtils.java

```diff
@@ -55,13 +57,13 @@ private DocsApiUtils() {}
    * @param path single filter or field path
    * @return Converted to represent expected path in DB, keeping the segmentation.
    */
-  public static String convertArrayPath(String path) {
+  public static String convertArrayPath(String path, int maxArrayLength) {
```

Comment:
do you think we should provide alternatives to the extended methods here that take the configuration as parameter?

```
convertArrayPath(String path, DocsApiConfiguraiton config) {
  return convertArrayPath(path, config.get..);
}
```